testsuite/MDAnalysisTests/data/coordinates/README.md

```diff
@@ -44,10 +44,7 @@ plain text using Gromacs utilities.
 test.h5md
 ---------
 ## Creation
-Converted test.trr to test.h5md with MDAnalysis and pyh5md
-using the 'create_h5md_data.py' script.
-See https://github.com/pdebuyl/pyh5md for pyh5md repository.
+Written with MDAnalysis using the 'create_h5md_data.py' script
```

Comment:
Does the h5md file now have good layout?

Reply:
yes, it's following the (1, n_atoms, 3) chunk scheme